### DISCUSSION TITLE
snippets/socketio-emit-cheatsheet

### DISCUSSION THREAD

**Discussion by giscus** [thutasann/thuta-portfolio#33] [Announcements]
# snippets/socketio-emit-cheatsheet

I'm a developer that loves building NEXTJS, ANGULAR & SpringBoot and IOS Mobile apps. Also loves building UI/UX with TailwindCSS, Material UI, RadixUI, etc

https://thutadev.vercel.app/snippets/socketio-emit-cheatsheet

**Comment by thutasann** [comment]
socket
